hugo-example/build.gradle

```diff
@@ -7,8 +7,8 @@ buildscript {
       mavenLocal()
     }
 
-    classpath 'com.android.tools.build:gradle:1.3.0'
-    classpath 'com.jakewharton.hugo:hugo-plugin:1.2.2-SNAPSHOT'
```

Comment:
The app module was not building with 1.2.2-SNAPSHOT.  It only built when I
changed it to the value pointed out in the README.
On Nov 7, 2015 6:57 AM, "Jake Wharton" <notifications@github.com> wrote:

> In hugo-example/build.gradle
> <https://github.com/JakeWharton/hugo/pull/105#discussion_r44211444>:
>
> > @@ -7,8 +7,8 @@ buildscript {
> >        mavenLocal()
> >      }
> >
> > -    classpath 'com.android.tools.build:gradle:1.3.0'
> > -    classpath 'com.jakewharton.hugo:hugo-plugin:1.2.2-SNAPSHOT'
>
> This was the correct value.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/JakeWharton/hugo/pull/105/files#r44211444>.
>